app/services/symphony_client.rb

```diff
@@ -64,16 +68,21 @@ def patron_info(patron_key, item_details: {})
       ].join(',')
     })
 
-    JSON.parse(response.body)
+    begin
+      JSON.parse(response)
+    rescue JSON::ParserError
+      nil
```

Comment:
I don't feel super strongly one way or the other, but do we want to know when/that this is occurring?  If this were to ever happen w/o us knowing, should we get a notification in Honeybadger?

Reply:
@shelleydoljack also mentioned this. My response to her was that currently we are not (and not really able) doing anything about it if it happens for some unknown SirsiDynix reason. Also when it happens every day during the Symphony restart do we want to get those alerts potentially every day? On the other hand, it might be good to have the data that is captured if someone complains. I can hook it up of you both want.